src/components/forms/ClaimForm.svelte

```diff
@@ -180,9 +180,12 @@ const setInitialValues = (claim: Claim, claimItem: ClaimItem) => {
   }
   lossReason = claim.incident_type || lossReason
   situationDescription = claim.incident_description || situationDescription
-  if (typeof claimItem.is_repairable === 'boolean') {
-    repairableSelection = claimItem.is_repairable ? 'repairable' : 'not_repairable'
-  }
+  repairableSelection =
+    typeof claimItem.is_repairable === 'boolean'
+      ? claimItem.is_repairable
+        ? 'repairable'
+        : 'not_repairable'
+      : 'repairable'
```

Comment:
i lover ternaries but nesting them can quickly become unmaintainable and hard to read. this could be refactored
```suggestion
  repairableSelection =
    typeof claimItem.is_repairable !== 'boolean' ||  claimItem.is_repairable
        ? 'repairable'
        : 'not_repairable'
```